### ISSUE TITLE
swcarpentry/python-novice-inflammation

### ISSUE THREAD

**Issue by zkamvar** [carpentries/lesson-transition#22]
This is an issue to track the parsing problems of swcarpentry/python-novice-inflammation as I build its R script. 

 - [x] alt text with two sets of quote types (https://github.com/carpentries/pegboard/issues/112) (fixed manually: https://github.com/carpentries/lesson-transition/blob/03110f63b4afe02faed540d250fd16a8b0c1e8bc/swcarpentry/python-novice-inflammation.R#L49-L52
 - [x] unparsed links in setup.md (fixed maunally via https://github.com/swcarpentry/python-novice-inflammation/pull/1031)
 - [x] hanging setup blocks in setup.md fixed manually: https://github.com/carpentries/lesson-transition/blob/03110f63b4afe02faed540d250fd16a8b0c1e8bc/swcarpentry/python-novice-inflammation.R#L54-L68
 - [x] missing episode links for 01-numpy and 04-files (changed to 02-numpy and 06-files)
 - [x] some python-generated image files (grid-{01,02,03}.png) get caught by the filter (fixed in https://github.com/carpentries/lesson-transition/commit/42c51ae882a5f6aa80cc9d870e26b1582a83283a):
       https://github.com/carpentries/lesson-transition/blob/20231912e36998894e5ff1f9ed5fdc6b2626dc8d/filter-list.txt#L68-L70 
 - [x] definition lists lose automatic linking. fixed manually: https://github.com/carpentries/lesson-transition/blob/03110f63b4afe02faed540d250fd16a8b0c1e8bc/swcarpentry/python-novice-inflammation.R#L39-L47

After the updates to the link processing (which invalidated the fixes I made earlier)

 - [x] missing file (04-files.html) moved to 06-files.html
 - [x] missing file (01-numpy.html) moved to 02-numpy.html

```markdown
episodes/11-debugging.md:93 [missing file]: [opening lesson](01-numpy.html)
episodes/12-cmdline.md:707 [missing file]: [earlier](04-files.html)
index.md:34 [missing file]: [](fig/lesson-overview.svg)
index.md:39 [missing file]: [comma-separated values](reference.html#comma-separated-values)
instructors/instructor-notes.md:102 [missing file]: [NumPy topic](01-numpy.html)
learners/discuss.md:310 [image missing alt-text]: fig/ipythonblocks_show_color_example.png
learners/reference.md:286 [missing anchor]: [floating-point](#float)
```

**Comment by zkamvar** [comment]
I have set up the repo at https://github.com/fishtree-attempt/python-novice-inflammation/

**Comment by zkamvar** [comment]
Hello @ineelhere and @noatgnu, I have created a test transition of your lesson and have invited you to collaborate on it on GitHub. 

Please browse through it and let me know here if there are any issues we need to fix before we set the date to transition your lesson next month. 

Thank you both!

**Comment by zkamvar** [comment]
The site is here: https://fishtree-attempt.github.io/python-novice-inflammation/

**Comment by zkamvar** [comment]
fixed in c54e338
